app/src/main/res/layout/choose_master_pswrd_fragment.xml

```diff
@@ -72,6 +72,29 @@
 
         </com.google.android.material.textfield.TextInputLayout>
 
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/add_a_hint"
```

Comment:
you can change this id to just "hint".
This is the convention that I try to follow : for TextInputLayout : "nameOfField" and for inner edit text : name of parent+"_text"